cmd/zpool/zpool.d/slaves

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Show device mapper slave devices.  This is useful for looking up the
+# /dev/sd* devices associated with a dm or multipath device.  For example:
+#
+# $ ls /sys/block/dm-113/slaves/
+# sddt  sdjw
+#
+
+if [ "$1" = "-h" ] ; then
```

Comment:
Ah, I just realized what you were doing in all the other scripts... and it's really clever. It's still a nit in the sense that you have a `helpstr` in most of the other scripts. But, I'm not so bothered by this anymore. Feel free to ignore the nits related to this.